omnibus/resources/agent/msi/cal/CustomAction.cpp

```diff
@@ -424,16 +424,20 @@ UINT doUninstallAs(MSIHANDLE hInstall, UNINSTALL_TYPE t)
     {
         std::string usershort;
         toMbcs(usershort, installedUser.c_str());
-        WcaLog(LOGMSG_STANDARD, "This install installed user %s, will remove", usershort.c_str());
+        WcaLog(LOGMSG_STANDARD, "This install installed user %s", usershort.c_str());
         installedUserPtr = installedUser.c_str();
         if (installState.getStringValue(installCreatedDDDomain.c_str(), installedDomain)) {
             installedDomainPtr = installedDomain.c_str();
             toMbcs(usershort, installedDomainPtr);
-            WcaLog(LOGMSG_STANDARD, "Removing user from domain %s", usershort);
+            WcaLog(LOGMSG_STANDARD, "NOT Removing user from domain %s", usershort);
```

Comment:
it's just a temporary variable for doing the log message, because the log API takes ascii and everything else is widechar.  I just reused the variable so I wouldn't have to allocate another.